examples/vision/3D_image_classification.py

```diff
@@ -0,0 +1,454 @@
+"""
+Title: 3D Image Classification from CT Scans
+Author: [Hasib Zunair](https://hasibzunair.github.io/)
+Date created: 2020/09/10
+Last modified: 2020/09/17
+Description: Train a 3D convolutional neural network to classify location of cancer.
+"""
+"""
+## Introduction
+
+This example will show the steps needed to build a 3D image classification
+model using a 3D convolutional neural network (CNN) to predict the location
+of cancerous regions in CT scans.
+
+## Why 3D CNNs at all?
```

Comment:
It would be useful to provide paper references here

Reply:
Done!